### DISCUSSION TITLE
Fancybox chage 'friction' and 'wheel' fail

### DISCUSSION THREAD

**Discussion by colormango** [fancyapps/ui#256] [Q&A]
https://jsfiddle.net/softcns/gdj1vf09/2/
I try to change 'friction' to 0.8 and change the 'wheel' to slide, but it still not working both.
<script>
	Fancybox.bind("[data-fancybox]", {
    Carousel :{
    friction: 0.8,
  },
  Image: {
    Panzoom: {
      zoomFriction: 0.7,
      wheel: slide,
      maxScale: function () {
        return 5;
      },
    },
  },
});
</script>

**Comment by fancyapps** [comment]
Hi,

First, your JavaScript code will not execute, because you have added `async` attribute to script tag. You can remove it.

Then your code syntax is not correct, it should be like this:

```js
  Fancybox.bind("[data-fancybox]", {
    Carousel :{
      friction: 0.8,
    },
    Image: {
      Panzoom: {
        zoomFriction: 0.7,
        wheel: slide,
        maxScale: function () {
          return 5;
        },
      },
    },
  });
```

https://jsfiddle.net/8etmn2b6/
